src/components/scenes/existingAccout/ChangeRecoveryScene.tsx

```diff
@@ -427,5 +415,14 @@ const getStyles = cacheStyles((theme: Theme) => ({
     marginBottom: theme.rem(1),
     fontSize: theme.rem(1),
     fontFamily: theme.fontFaceDefault
+  },
+  messageText: {
+    color: theme.primaryText,
+    fontFamily: theme.fontFaceDefault,
+    fontSize: theme.rem(1),
+    marginVertical: theme.rem(0.5)
+  },
```

Comment:
I believe this style is unused now.

Reply:
Removed [here](https://github.com/EdgeApp/edge-login-ui-rn/pull/55/commits/f63953f9b8af1772da3e837a18c992be5a03d404) - no `!fixup` commit for this one.